sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1417,7 +1417,12 @@ def _add_random_shard(element):
       value = element[1]
       return ((key, random.randint(0, DEFAULT_SHARDS_PER_DESTINATION)), value)
 
-    def _to_hashable_table_ref(table_ref_elem_kv):
+    V = TypeVar('V')
+    TABLE_REF_TYPE = TypeVar('TABLE_REF_TYPE')
```

Comment:
Sorry I was lazy when I wrote TABLE_REF_TYPE - I meant it should be replaced with `bigquery.TableReference`. (the Union should contain the types accepted by bigquery_tools.get_hashable_destination)

Reply:
Aha. That makes more sense! Corrected.